cogs/server.py

```diff
@@ -248,6 +248,19 @@ def __init__(self, bot):
 
         await ctx.send("Ticket successfully closed!")
 
+    @commands.command(aliases=["sexauth","sexternalauth"])
+    @auth.check_auths([AuthPerms.R_ADMIN])
+    async def external_auth(self, ctx, state: int):
```

Comment:
Base Command name is not in line with the other ones.
Should be server_external_auth